src/contingency_model/contingency_constraints.jl

```diff
@@ -208,3 +208,75 @@ function add_event_constraints!(
     end
     return
 end
+
+function add_reactive_power_contingency_constraint(
+    container::OptimizationContainer,
+    ::Type{T},
+    ::Type{R},
+    ::Type{P},
+    devices::Union{Vector{V}, IS.FlattenIteratorWrapper{V}},
+    model::DeviceModel{V, W},
+    ::Type{X},
+) where {
+    T <: ConstraintType,
+    R <: VariableType,
+    P <: ParameterType,
```

Comment:
Can't we make this more concrete? We know exactly which parameters, variables and constraints they are.